src/main/java/gitflow/actions/RepoActions.java

```diff
@@ -144,11 +143,16 @@
                 actionList.add(new FinishHotfixAction(myRepo));
 
                 //can't publish hotfix if it's already published
-                if (branchUtil.isCurrentBranchPublished() == false) {
+                if (!branchUtil.isCurrentBranchPublished()) {
```

Comment:
Please retain the existing conditional formatting. It's more verbose for readability purposes